README.md

```diff
@@ -7,6 +7,8 @@
 akka-persistence-jdbc writes journal and snapshot entries to a configured JDBC store. It implements the full akka-persistence-query API and is therefore very useful for implementing DDD-style 
 application models using Akka for creating reactive applications.
 
+Please note that the H2 database is not recommended to be used as a production database, and support for H2 is primarily for testing purposes.
+
```

Comment:
Could we have that same sentence in that page? 
https://doc.akka.io/docs/akka-persistence-jdbc/current/configuration.html 

We can use a callout just after the list of supported databases: 
https://developer.lightbend.com/docs/paradox/current/directives/callouts.html